contracts/Fragment.sol

```diff
@@ -264,10 +270,17 @@ contract Fragment is
         auths[0].remove(addr);
     }
 
+    /// @notice Attaches a Proto-Fragment from Fragnova's Blockchain to this Smart Contract
+    /// @param fragmentHash - The Proto-Fragment ID to attach
+    /// @param signature - The signature provided by Fragnova's Blockahin's Off-Chain Validator to validate this attach request
+    /// @dev Verifies if the authorizer signed the message.
+    /// Then, it  mints an ERC-721 Token (where the ID is the one plus the uint256 stored in `uint256(keccak256(abi.encodePacked(FRAGMENT_COUNTER)))`)
+    /// and gives its ownership to `msg.sender`
     function attach(bytes32 fragmentHash, bytes calldata signature) external {
         require(!_exists(uint256(fragmentHash)), "Fragment already attached");
 
         uint64[1] storage nonce;
+        // ¿I wonder why we there is only one authorizer?
```

Comment:
it's not actually, this is a hack to access unstructured data from Assembly, as you can see I do for everything.

Reply:
Oh you are right. It's an array of AddressSets. So the AddressSet can have multiple authorizers. Thanks!